### PR TITLE
add Merkle DAG tutorial announcement

### PR DESCRIPTION
**Pull request by terichadbourne**
DRAFT WIP - First draft of a tutorial announcing our latest ProtoSchool tutorial. Still needs some image edits but ready for content review. 

To Do: 
- [x] get some proofs
- [x] update images / captions
- [x] fix dates
- [x] ~DO NOT PUBLISH until~ tutorial has launched :)

### CODE REVIEW

**Comment by jennwrites** [comment]
Stoked about this @terichadbourne! I will copyedit tomorrow, Jan 13!

File: content/post/116-protoschool-merkle-dags.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+<img src="/116-protoschool-merkle-dags/dag-deduplication.png" alt="Merkle DAG diagram"/>
+
+_Screenshot of a Merkle DAG from the new tutorial_
+
+In IPFS, as across the decentralized web, content addressing enables us to securely locate and identify data stored by peers. The Content Identifiers (CIDs) we often use to link to individual files or pieces of data can also be used to express complex webs of data in their entirety, when we structure that data as content-addressable Merkle DAGs. Merkle DAGs, wrapped in structures provided by IPLD, are key to the interoperability of IPFS, libp2p, Filecoin, and other distributed protocols. In this new code-free ProtoSchool tutorial, we take a deep dive into the properties that make Merkle DAGs the backbone of the distributed web, and explore the many benefits that the data structure unlocks, from verifiability to distributability to deduplication. [Check it out!](https://proto.school/merkle-dags)
```
Comment (mitchwagner) [inline content/post/116-protoschool-merkle-dags.md:18]: "Merkle DAGs, wrapped in structures..." doesn't quite sound right to my ear. IPLD defines a structure for a Merkle DAG, more than than wrapping it.

Perhaps something closer to "Merkle DAGs, structured using schemas provided by IPLD..."?

Comment (terichadbourne) [reply content/post/116-protoschool-merkle-dags.md]: Thanks for the suggestion! Wasn't sure how to describe that relationship.

File: content/post/116-protoschool-merkle-dags.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+date: 2021-01-10
+url: /2021-01-14-protoschool-tutorial-merkle-dags/
+title: Explore Merkle DAGs at ProtoSchool
+author: Teri Chadbourne
+tags: ipfs, ipld, protoschool
+header_image: 116-protoschool-merkle-dags.png
+---
+
+The [ProtoSchool](https://proto.school) team is pleased to announce the launch of a shiny new multiple-choice tutorial on [Merkle DAGs](https://proto.school/merkle-dags), the cornerstone of our freshly launched [IPLD course](https;//proto.school/course/ipld). 🎉
```
Comment (jennwrites) [inline content/post/116-protoschool-merkle-dags.md:10]: I'm still a big fan of spelling out acronyms on first reference, so in this case I'd like to see IPLD spelled out in this intro paragraph.

**Review by jennwrites — APPROVED** [review]
This looks great, pending the completion of the to-dos. Great work Proto-folks!

File: content/post/116-protoschool-merkle-dags.md (hunk 2 above)
Comment (terichadbourne) [reply content/post/116-protoschool-merkle-dags.md]: Good call. Done!

**Comment by jennwrites** [comment]
I had tentatively scheduled this for publication today, but what's the timeline like on your end @terichadbourne?

**Comment by terichadbourne** [comment]
> I had tentatively scheduled this for publication today, but what's the timeline like on your end @terichadbourne?

@jennwrites Looking hopeful for this afternoon, just doing a couple of final checks on the tutorial and fixing a build bug. Will keep you posted! If we miss today, I was thinking Tuesday might be the next reasonable option given the holiday weekend.

**Comment by terichadbourne** [comment]
I've changed the dates today in hopes we get the tutorial published successfully. 

@jennwrites I built this in a `116-whatever` folder because I saw that the Igalia PR was using `115-` and I'd cause a merge conflict if I went sequentially. Do you want me to bother fixing, if you think mine is going first?

**Comment by jennwrites** [comment]
I can fix if we post today! Otherwise, if it goes out on Tues, then we'll be in sequential order ☺️ since Igalia publishes tomorrow for sure.

**Comment by terichadbourne** [comment]
@jennwrites We got the site updated, so you can go ahead and swap the folder names and publish. Thanks so much for your help!

**Pull request merged**